tests/functional/UseCase/Table/Import/FromTable/IncrementalImportTableFromTableTest.php

```diff
@@ -178,16 +178,6 @@ public function importTableFromTableFullLoadWithTimestampTableWithTypesProvider(
     {
         yield 'no feature' => [
             'features' => [],
-            'nOfUpdatedTimestamps' => 3,
```

Comment:
to je kvoli tomu updatu co robil Jirka? Alebo preco to predtim preslo?

Reply:
jj kvůli tomu co dělal jirka, předtím sa to tak chovalo na featuru teď je to default.